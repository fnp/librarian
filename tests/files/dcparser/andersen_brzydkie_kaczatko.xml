<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1/">
    <rdf:Description rdf:about="http://redakcja.wolnelektury.pl/documents/book/brzydkie-kaczatko/">
        <dc:creator xml:lang="pl">Andersen, Hans Christian</dc:creator>
        <dc:title xml:lang="pl">Brzydkie kaczątko</dc:title>
        <dc:contributor.translator xml:lang="pl">Niewiadomska, Cecylia</dc:contributor.translator>
        <dc:contributor.technical_editor xml:lang="pl">Gałecki, Dariusz</dc:contributor.technical_editor>
        <dc:publisher xml:lang="pl">Fundacja Wolne Lektury</dc:publisher>
        <dc:subject.period xml:lang="pl">Romantyzm</dc:subject.period>
        <dc:subject.type xml:lang="pl">Epika</dc:subject.type>
        <dc:subject.genre xml:lang="pl">Baśń</dc:subject.genre>
        <dc:description xml:lang="pl">Publikacja zrealizowana w ramach projektu Wolne Lektury (http://wolnelektury.pl). Reprodukcja cyfrowa wykonana przez Bibliotekę Narodową z egzemplarza pochodzącego ze zbiorów BN.</dc:description>
        <dc:identifier.url xml:lang="pl">http://wolnelektury.pl/katalog/lektura/brzydkie-kaczatko</dc:identifier.url>
        <dc:source.URL xml:lang="pl">http://www.polona.pl/dlibra/doccontent2?id=3563&amp;dirids=4</dc:source.URL>
        <dc:source xml:lang="pl">Andersen, Hans Christian (1805-1875), Baśnie, Gebethner i Wolff, wyd. 7, Kraków, 1925</dc:source>
        <dc:rights xml:lang="pl">Domena publiczna - tłumacz Cecylia Niewiadomska zm. 1925</dc:rights>
        <dc:date.pd xml:lang="pl">1925</dc:date.pd>
        <dc:format xml:lang="pl">xml</dc:format>
        <dc:type xml:lang="pl">text</dc:type>
        <dc:type xml:lang="en">text</dc:type>
        <dc:date xml:lang="pl">2007-08-14</dc:date>
        <dc:audience xml:lang="pl">SP1</dc:audience>
        <dc:language xml:lang="pl">pol</dc:language>
    </rdf:Description>
</rdf:RDF>
