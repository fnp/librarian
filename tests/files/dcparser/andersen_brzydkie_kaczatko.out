{
    'publisher': ['Fundacja Wolne Lektury'],
    'about': 'http://redakcja.wolnelektury.pl/documents/book/brzydkie-kaczatko/',
    'source_name': 'Andersen, Hans Christian (1805-1875), Baśnie, Gebethner i Wolff, wyd. 7, Kraków, 1925',
    'author': 'Andersen, Hans Christian',
    'url': 'http://wolnelektury.pl/katalog/lektura/brzydkie-kaczatko',
    'created_at': '2007-08-14',
    'title': 'Brzydkie kaczątko',
    'kind': 'Epika',
    'source_url': 'http://www.polona.pl/dlibra/doccontent2?id=3563&dirids=4',
    'translators': ['Niewiadomska, Cecylia'],
    'released_to_public_domain_at': '1925',
    'epoch': 'Romantyzm',
    'genre': 'Baśń',
    'technical_editors': ['Gałecki, Dariusz'],
    'license_description': 'Domena publiczna - tłumacz Cecylia Niewiadomska zm. 1925',
    'audiences': ['SP1'],
    'language': 'pol',
}
